src/Parsers/HiliteComparator/compare_hilites.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+model_formatter="/home/natasha/MurfelClickHouse/build/programs/clickhouse-format";
+new_formatter="/home/natasha/copyClickHouse/build/programs/clickhouse-format";
+compare="/home/natasha/copyClickHouse/tests/a.out"
```

Comment:
Please change this line into

```
hilite_comparator="../../../build/src/Parsers/HiliteComparator/hilite_comparator"
```

and write a comment that in order to use this script it's necessary to first build this comparator along with `clickhouse-format` and that can be performed with commands like `cd build && ninja clickhouse-format hilite_comparator`.

Reply:
I've written hilite_comparator as an argument as well:
```
model_formatter=$1  # e.g. "ClickHouse/build/programs/clickhouse-format";
new_formatter=$2  # e.g. "refactorClickHouse/build/programs/clickhouse-format";
hilite_comparator=$3  # e.g. "refactorClickHouse/build/src/Parsers/HiliteComparator/hilite_comparator"
```

Instead of the relative path you've suggested, because we cannot assume where the bash script would be called from (as in 'which repo', and also as in which folder).